src/scripts/site.js

```diff
@@ -62,7 +62,8 @@ var Dashboard = {
         },
         logout: function(logoutWithServer) {
             function onLogoutDone() {
-                Dashboard.navigate("login.html")
+                var loginPage;
+                AppInfo.isNativeApp ? (loginPage = "selectserver.html", window.ApiClient = null) : loginPage = "login.html", Dashboard.navigate(loginPage)
```

Comment:
can we do less poetic code please? :) like
```suggestion
                if (AppInfo.isNativeApp) {
                    loginPage = "selectserver.html";
                    window.ApiClient = null;
                } else {
                    loginPage = "login.html";
                }
                Dashboard.navigate(loginPage);
```

Reply:
good change :) this change was actually copied over from an old working version so I didn't think to update the readability